tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/TspClientTool/TspClientTool.cs

```diff
@@ -0,0 +1,476 @@
+                repoOpt,
+                commitOpt,
+                tspConfigOpt,
+                localSpecRepoOpt,
+                emitterOptionsOpt,
+                saveInputsOpt,
+                skipInstallOpt,
+                traceOpt
+            };
+
+            var syncCommand = new Command(syncCommandName, "Sync TypeSpec project specified in tsp-location.yaml")
+            {
+                localSpecRepoOpt
+            };
+
+            var generateCommand = new Command(generateCommandName, "Generate from a TypeSpec project")
+            {
+                emitterOptionsOpt,
+                saveInputsOpt,
+                skipInstallOpt,
+                traceOpt
+            };
+
+            // Set handlers
+            initCommand.SetHandler(async ctx => await HandleCommand(ctx, ctx.GetCancellationToken()));
+            updateCommand.SetHandler(async ctx => await HandleCommand(ctx, ctx.GetCancellationToken()));
+            syncCommand.SetHandler(async ctx => await HandleCommand(ctx, ctx.GetCancellationToken()));
+            generateCommand.SetHandler(async ctx => await HandleCommand(ctx, ctx.GetCancellationToken()));
+
+            // Add subcommands to root
+            rootCommand.AddCommand(initCommand);
+            rootCommand.AddCommand(updateCommand);
+            rootCommand.AddCommand(syncCommand);
+            rootCommand.AddCommand(generateCommand);
+
+            return rootCommand;
+        }
+
+        public override async Task HandleCommand(InvocationContext ctx, CancellationToken ct)
+        {
+            try
+            {
+                var commandResult = ctx.ParseResult.CommandResult;
+                var commandName = commandResult.Command.Name;
+
+                logger.LogInformation("Executing tsp-client command: {CommandName}", commandName);
+
+                var response = commandName switch
+                {
+                    initCommandName => await ExecuteInitCommand(ctx),
+                    updateCommandName => await ExecuteUpdateCommand(ctx),
+                    syncCommandName => await ExecuteSyncCommand(ctx),
+                    generateCommandName => await ExecuteGenerateCommand(ctx),
+                    _ => new DefaultCommandResponse { ResponseError = $"Unknown command: {commandName}" }
+                };
+
+                ctx.ExitCode = ExitCode;
+                output.Output(response);
+            }
+            catch (Exception ex)
+            {
+                logger.LogError(ex, "Error executing tsp-client command");
+                SetFailure(1);
+                ctx.ExitCode = ExitCode;
+                output.Output(new DefaultCommandResponse
+                {
+                    ResponseError = $"Error executing tsp-client command: {ex.Message}"
+                });
+            }
+
+            await Task.CompletedTask;
+        }
+
+        [McpServerTool(Name = "tsp-client-init"), Description("Initialize SDK project folder from tspconfig.yaml")]
```

Comment:
Description says, initialize SDK project. Is it better to rename tool name to make it more `azsdk.init-sdk-project`
My concern is that tsp-client-init is conveying it's use only for those who know they need to run tsp init to start